README.md

```diff
@@ -56,3 +56,44 @@ of our [mailing lists](https://netflix.github.io/mantis/community/#mailing-lists
 For more information on contribution, check out the contributions file here:
 
 - [https://github.com/Netflix/mantis/blob/master/CONTRIBUTING.md](https://github.com/Netflix/mantis/blob/master/CONTRIBUTING.md)
+
+### Module Structure
+This excludes all connectors and examples as they are mostly leaf nodes in the dependency graph. 
+
+module                      | purpose                                                                                                               | examples                                                                   | packageprefixes
+----------------------------|-----------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------
+mantis-common-serde         | json serializer/deserializer                                                                                          | JsonSerializer                                                             | io.mantisrx.common.*
+mantis-discovery-proto      | not sure                                                                                                              | JobDiscoveryProto,AppJobClustersMap,StageWorkers                           | com.netflix.mantis.discovery.proto.*
```

Comment:
how about adding this to something `<need to fill this in>` to make it look a little better.
it'd be nice to have consistent naming/capitalization experience.
eg: `common`, `client` and `Runtime`, `Contains`